docs/api.md

```diff
@@ -358,6 +358,7 @@ Shortcut for [page.mainFrame().addScriptTag(url)](#frameaddscripttagurl).
 - `options` <[Object]>
   - `button` <[string]> `left`, `right`, or `middle`, defaults to `left`.
```

Comment:
What does left, right, middle refer to?

Reply:
@gajus if I understand correctly, it's the mouse buttons - `left button click`, `right button click` and `middle button click`

Reply:
I see. I guess it was not obvious since I haven't used a mouse in the last +5 years. :-)